wp-content/plugins/core/src/Post_Types/Page/Config.php

```diff
@@ -0,0 +1,24 @@
+<?php declare(strict_types=1);
+
+namespace Tribe\Plugin\Post_Types\Page;
+
+class Config {
+
+	protected string $post_type = Page::NAME;
+
+	/**
+	 * Define block templates for initial editor state
+	 */
+	public function register_block_template(): void {
+		$post_type_object           = get_post_type_object( Page::NAME );
+		$post_type_object->template = [
+			[
+				'core/pattern',
+				[
+					'slug' => 'patterns/page-placeholder',
```

Comment:
Question: I wonder if we just rename this to `pattern/page` (avoid placeholder) since more than likely a dev will make modifications to the pattern as the default (rather than replacing the pattern with a different file) for each project.